File: SelectionSort.py
#SelectionSort
def selection_sort(array):
    for i in range(len(array)):
        min = i

        for j in range(i+1, len(array)):
            if array[min] > array[j]:
                min = j        
       
        array[i], array[min] = array[min], array[i]
    
data = [6,5,3,4,1,2,7,8]
selection_sort(data)
print(data)

File: SelectionSort.js
//SelectionSort
function selectionSort(array){
    for(var i = 0; i < array.length; i++){
        let min = i;

        for(let j = i + 1; j < array.length; j++){
            if(array[min] > array[j]){
                min = j;
            }
        }
        
        if(i !== min){
            let x = array[i];
            array[i] = array[min]
            array[min] = x
        }
    }
}

data = [6,3,5,1,2,7,4,8];
selectionSort(data);
console.log(data);